logux-vue-todomvc/src/App.vue

```diff
@@ -0,0 +1,406 @@
+<template>
+  <section class="todoapp">
+    <!-- header -->
+    <header class="header">
+      <h1>todos</h1>
+      <input class="new-todo"
+        autofocus
+        autocomplete="off"
+        placeholder="What needs to be done?"
+        @keyup.enter="addTodo">
+    </header>
+    <!-- main section -->
+    <section class="main" v-show="todos.length">
+      <input class="toggle-all" id="toggle-all"
+        type="checkbox"
+        :checked="allChecked"
+        @change="toggleAll(!allChecked)">
+      <label for="toggle-all"></label>
+      <ul class="todo-list">
+        <TodoItem
+          v-for="(todo, index) in todos"
+          :key="index"
+          :todo="todo"
+        />
+      </ul>
+    </section>
+    <!-- footer -->
+    <footer class="footer" v-show="todos.length">
+      <span class="todo-count">
+        <strong>{{ remaining }}</strong>
+        {{ remaining | pluralize('item') }} left
+      </span>
+    </footer>
+  </section>
+</template>
+
+<script>
+import TodoItem from './components/TodoItem.vue'
+
+export default {
+  components: { TodoItem },
+  data () {
+    return {}
+  },
+  computed: {
+    todos () {
+      return this.$store.state.todos
+    },
+    allChecked () {
+      return this.todos.every(todo => todo.done)
+    },
+    remaining () {
+      return this.todos.filter(todo => !todo.done).length
+    }
+  },
+  methods: {
+	toggleAll () {
+		this.todos
```

Comment:
You mixed tabs and spaces.

Add `.editorconfig` from other Logux projects. I also recommend to use Logux ESLint config.